myuw/data/resource_link_import.csv

```diff
@@ -76,7 +76,7 @@ Campus Life,Getting Around Campus,all,,,https://facilities.uw.edu/transportation
 Campus Life,Getting Around Campus,all,,,http://uwshuttles.com/,UW Shuttles,,,,,no
 Campus Life,Health & Wellness,all,https://wellbeing.uw.edu/,Husky Health & Well-Being,,,,,,,no
 Campus Life,Health & Wellness,all,,,http://www.washington.edu/counseling/,Counseling Center,http://www.uwb.edu/studentservices/counseling,Counseling Services,http://www.tacoma.uw.edu/node/44255,Student Counseling Center,no
-Campus Life,Health & Wellness,all,,,http://depts.washington.edu/hhpccweb/,Hall Health Center,http://www.uwb.edu/recwell/health-promotion,Health Promotion,http://www.tacoma.uw.edu/student-health-services/about-student-health-services,Student Health Services (SHS),no
+Campus Life,Health & Wellness,all,,,http://depts.washington.edu/hhpccweb/,Hall Health Center,https://www.uwb.edu/recwell,Health Promotion,http://www.tacoma.uw.edu/student-health-services/about-student-health-services,Student Health Services (SHS),no
```

Comment:
Looks like UWB has a new wellness center:
Health and Wellness Resource Center - https://www.uwb.edu/arc/hawrc